_posts/guides/2021-12-21-guide_gamemode-info-surf.md

```diff
@@ -27,4 +27,7 @@ Surf is a very popular gamemode
 
 ## How Surf Works
 
-TODO
\ No newline at end of file
+You can surf by holding A on the right side of the ramp and D on the left without holding W.
```

Comment:
Let's change "You can surf" to something like "Surfing on a ramp requries". Comma between "left" and "without".